akka-cluster/src/main/resources/reference.conf

```diff
@@ -289,5 +289,10 @@ akka {
       adaptive-group = "akka.cluster.routing.AdaptiveLoadBalancingGroup"
     }
   }
+  
+  coordinated-shutdown-phases {
+    cluster-pre-exiting = []
+    cluster-exiting = [cluster-pre-exiting]
+  }
```

Comment:
the approach outlined by @ktoso also allows us to add readable names to the phases:

```
cluster-pre-exiting {
  description = "Performing tasks before cluster Exit"
}
```

or something like that.